pytest_tests/lib/helpers/http_gate.py

```diff
@@ -46,7 +46,7 @@ def get_via_rest_gate(
     if download:
         download_attribute = "?download=true"
     if request_path is None:
-        request = f"{endpoint}/get/{cid}/{oid}{download_attribute}"
+        request = f"{endpoint}/objects/{cid}/by_id/{oid}{download_attribute}"
```

Comment:
Do we have tests for old APIs after that?

Reply:
We have tests for the old upload function, but not for `/get*`. I know an easy way to add some tests for object download, and I'll take care of that.

Reply:
OK, I want old APIs to remain being tested, they'll be kept for a long time, so we shouldn't throw away tests that we have for them.

Reply:
Done.